prep-javascript-dom-events/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JavaScript Events</title>
+  </head>
+
+  <body>
+    <button class="hello-world-btn">Hello World!</button>
+    <script>
+      // Your code here
+      document.addEventListener(
+        'DOMContentLoaded',
+        function processUserInput(callback) {}
+      );
+      console.log('content loaded!');
+
+      window.addEventListener(
+        'mousemove',
+        function processUserInput(callback) {}
+      );
+      console.log('mouse is moving!');
+
+      var $helloWorld = document.querySelector('hello-world-btn');
+      $helloWorld.addEventListener(
+        'click',
+        function processUserInput(callback) {
+          'hello world!';
+        }
+      );
+      console.log('hello world!');
```

Comment:
Try moving your console logs into your function code blocks so that the console logs don't run until the events happen, like the user moving their mouse or clicking the button